packages/nextjs/local_database/importSeed.ts

```diff
@@ -0,0 +1,13 @@
+// import fs from "fs";
+
+export async function importSeed() {
+  //   console.log("jdjjdjd");
```

Comment:
Why is this function commented on?

Reply:
I aimed to develop a function that reads the project.json file and uses its data to seed the local database with projects. This way, the front-end team will have easy access to relevant data for their work.